lib/rtcpeerconnection/firefox.js

```diff
@@ -220,20 +220,6 @@ Object.defineProperty(FirefoxRTCPeerConnection.prototype, 'peerIdentity', {
 // Bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1133874
 FirefoxRTCPeerConnection.prototype.removeStream = function removeStream() {};
 
-// NOTE(mmalavalli): We implement addStream() by calling
-// RTCPeerConnection#addTrack() on each of the MediaStreamTracks in the
-// given MediaStream that have not already been added.
-FirefoxRTCPeerConnection.prototype.addStream = function addStream(mediaStream) {
-  var localTracks = this._peerConnection.getSenders().map(function(sender) {
-    return sender.track;
-  });
-  var newLocalTracks = util.difference(mediaStream.getTracks(), localTracks);
-
-  newLocalTracks.forEach(function(mediaStreamTrack) {
-    this._peerConnection.addTrack(mediaStreamTrack, mediaStream);
-  }, this);
-};
-
```

Comment:
Any harm to leaving this as-is?

Reply:
@markandrus Why would we need to retain this any longer?

Reply:
I guess you're right 👍